docker-compose.yml

```diff
@@ -1,14 +1,34 @@
 version: "3"
 
+secrets:
+  postgres_password:
+    file: ./secrets/postgres_password.txt
+  firebase_admin_service_account:
+    file: ./secrets/firebase_admin_service_account.json
+
 services:
+  postgres-db:
+    image: postgres
+    ports:
+      - "5431:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_DB: peerprepdb
+      POSTGRES_PASSWORD: /run/secrets/postgres_password
+    secrets:
+      - postgres_password
```

Comment:
technically we don't need this if we use a serverless DB - unless you have it for local testing?

Reply:
I was using this for local testing